run_create_appstream_fleet_autoscale.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+
+import json
+import time
+
+from env import env
+from run_common import AWSCli
+
+if __name__ == "__main__":
+    from run_common import parse_args
+
+    args = parse_args()
+
+    aws_cli = AWSCli()
+
+    env = env['appstream']['STACK'][0]
+    fleet_name = f'fleet/{env["FLEET_NAME"]}'
+    max_capacity = f'{env["MIN_CAPACITY"]}'
+    min_capacity = f'{env["MAX_CAPACITY"]}'
```

Comment:
@GoldieKMG // 여기서 string으로 치환할 필요는 없을것 같아요 그리고 min, max 바꿔쓰신것 같습니다.
```suggestion
    min_capacity = env["MIN_CAPACITY"]
    max_capacity = env["MAX_CAPACITY"]
```

Reply:
치환하면서 위아래로 잘못 넣어줫네요 .. 감사합니다!!